### PR TITLE
Brendan Shaeffer | Incomplete

### PR DESCRIPTION
**Pull request by toastyThink**
I unfortunately didn't get to all of the prompts in the homework. Prompt 11 gave me a lot of headaches. I don't think I solved it right but after today's lesson I think I understand how to find the median of an array. I will have to revisit this at a later date.

Also there is a typo in one of the commits. "Prompt y Complete"  should be "Prompt 7 Complete.

### CODE REVIEW

**Comment by toastyThink** [comment]
These are my answers for prompts 2 - 11.

File: lib/challenge.js [hunk 1]
```diff
@@ -134,19 +190,31 @@ function prompt7() {
...
 
-// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
-// prompt8();
+//          }else if(someNumber >= 30 && someNumber <= 60){
+//            console.log("The number is medium sized." + someNumber);
+
+//              }else if(someNumber > 60 && someNumber < 100){
+//                console.log("We got a big one!" + someNumber);
+//                }
+//    }
```
Comment (krabecb) [inline lib/challenge.js:212]: I believe indentation is off here and throughout challenge.js 🤔

File: lib/challenge.js [hunk 2]
```diff
@@ -177,18 +254,25 @@ function prompt9() {
...
-  const myArray = [];
+//console.log('prompt 10 begins here');
+
+//function prompt10() {
+//  const myArray = [];
   // YOUR CODE HERE
+ //   for(let i =1; i<100; ++i){
+ //     myArray.push(i); 
+
+ //   }
```
Comment (krabecb) [inline lib/challenge.js:265]: I would keep your vertical spacing consistent!

**Review by krabecb — COMMENTED** [review]
Overall, great work on this 💯 I would say make sure to keep your indentation and vertical spacing consistent for readability. Great work on committing often! Closing PR and marking "complete".

**Pull request closed without merging**